scripts/create-index-pages.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const logger = require('./utils/logger');
+const { BASE_DIR } = require('./constants');
+const { resolve } = require('path');
+
+const getTitle = (file) =>
+  file
+    .split('/')
+    .slice(-1)[0]
+    .replace('.mdx', '')
+    .split('-')
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ');
+
+const getFrontmatter = (title) => `---
+title: ${title}
+template: basicDoc
+---
+
+`;
+
+const getPageContent = (dir, files) => {
+  return files
+    .map((file) => {
+      const path = `${dir}/${file.name}`.replace(BASE_DIR, '');
+      // TODO: get the label from frontmatter
+      const label = getTitle(file.name);
```

Comment:
At the moment, this takes the last part of the filepath and turns it into a label. We probably want to grab the `title` from the frontmatter for that file and use that instead. If someone wants to take a crack at this, that would be rad.

Reply:
I'm working on this!